extensions/reclaim/src/hooks/useApi.tsx

```diff
@@ -0,0 +1,19 @@
+import { Detail, getPreferenceValues, useNavigation } from "@raycast/api";
+import { NativePreferences } from "../types/preferences";
+import { fetcher } from "../utils/axiosPromise";
+
+const useApi = () => {
+  const { apiToken } = getPreferenceValues<NativePreferences>();
+
+  const { push } = useNavigation();
+
+  if (!apiToken) {
+    push(
+      <Detail markdown={"Something wrong with your API Token key. Check your raycast config and set up a new token."} />
```

Comment:
Capitalised Raycast
```suggestion
      <Detail markdown={"Something wrong with your API Token key. Check your Raycast config and set up a new token."} />
```

Reply:
You can also add some `actions` to this `Detail` that [opens the Preferences](https://developers.raycast.com/api-reference/preferences#opencommandpreferences)

```
<Action
      title="Open Preferences"
      onAction={openCommandPreferences}
/>
```

Reply:
Nice!